script/release

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd "$(dirname "$0")/.."
+
+prev=$(git describe --abbrev=0 --tags)
+
+# Run checks, bump version. Create a commit, tag and push.
+yarn np \
+  --test-script checks \
+  --no-release-draft \
+  --no-publish
+
+# Build artifacts.
+yarn bundle:chrome
+yarn bundle:firefox
+
+# # Create a GitHub release, upload build artifacts.
+notes=$(./script/list-changes "$prev")
+tag=$(git describe --abbrev=0 --tags)
+gh release create "$tag" \
+  --notes "$notes" \
+  ./dist/*.zip
```

Comment:
`gh create` has an an interactive mode which allows to generate the release notes or use the ones on the git tag. By skipping `-- notes`, we could make use of that 🤔